.github/workflows/swarco_itc3.yml

```diff
@@ -0,0 +1,48 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# This workflow will run rspec
+
+name: Swarco ITC-3
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  workflow_dispatch:
+
+jobs:
+  test:
+
+    runs-on: [ self-hosted, Windows, X64, itc3 ]
```

Comment:
how come itc3 checks are hosted on windows while itc2 are on linux?

Reply:
Swarco prefere Windows. It also provides remote access to the configuration tool for the ITC-3.